mex/common/wikidata/convenience.py

```diff
@@ -1,71 +0,0 @@
-from collections.abc import Callable, Iterable
```

Comment:
the renaming of this file should go into the changelog as `BREAKING: ...`

Reply:
Done